BP/common/decisions/BP_government_conversion_decisions.txt

```diff
@@ -0,0 +1,402 @@
+	}
+
+	is_valid = {
+		custom_description = {
+			text = faith_is_reformed
+			subject = root
+			faith = { NOT = { has_doctrine_parameter = unreformed } }
+		}
+		is_independent_ruler = yes #Must be independent
+		highest_held_title_tier = 5 #Must be an empire
+		prestige_level >= 4
+		capital_province.county = { development_level >= convert_to_bureaucratic_development_requirement }
+		
+		#Tribal Innovations
+		custom_description = {
+			text = text_has_entire_tribal_innovations
+			subject = root.culture
+			culture = {
+				#All valid tribal innovations
+				has_innovation = innovation_vexilium
+				has_innovation = innovation_mustering_grounds
+				has_innovation = innovation_castrae
+				has_innovation = innovation_quilted_armor
+				has_innovation = innovation_development_00
+				has_innovation = innovation_leges_barbarorum
+				has_innovation = innovation_writing
+				has_innovation = innovation_barter
+				has_innovation = innovation_oppidum
+			}
+		}
+		
+		#Late Antiquity Innovations
+		custom_description = {
+			text = text_has_entire_late_antiquity_innovations
+			subject = root.culture
+			culture = {
+				#All valid tribal innovations
+				has_innovation = innovation_standing_army
+				has_innovation = innovation_limitanei
+				has_innovation = innovation_catapult
+				has_innovation = innovation_mail_armor
+				has_innovation = innovation_muri
+				has_innovation = innovation_conscription
+				has_innovation = innovation_development_01
+				has_innovation = innovation_fora
+				has_innovation = innovation_coinage
+				has_innovation = innovation_mundium
+				has_innovation = innovation_state_religion
+			}
+		}
+		
+		#Early Migration
+		custom_description = { #You need to have ALL (non-regional/unique) tribal innovations and roman heritage
+			text = text_has_required_early_migration_innovations
+			subject = root.culture
+			culture = {
+				#Required Early Migration Innovations
+				has_innovation = innovation_migratory_tactics
+				has_innovation = innovation_migratory_defenses
+				has_innovation = innovation_double_entry_bookkeeping
+				has_innovation = innovation_tenant_farming
+				has_innovation = innovation_development_02
+				has_innovation = innovation_doctrine_and_dogma
+			}
+		}
+		#Late Migration Era Innovation
+		culture = { has_innovation = innovation_civil_service_revitalization }
+	}
+
+	cost = { prestige = 250 }
+
+	effect = {
+		change_government = bureaucratic_government
+		add_realm_law_skip_effects = bureaucratic_authority_2
+		add_realm_law_skip_effects = imperial_elective_succession_law
+	}
+
+	ai_potential = { always = yes }
+
+	ai_will_do = { base = 100 }
+}
+
+convert_to_feudalism_decision = {
+	picture = "gfx/interface/illustrations/decisions/decision_realm.dds"
+	major = yes
+	desc = convert_to_feudalism_decision_desc
+
+	ai_check_interval = 36
+
+	is_shown = {
+		OR = {
+			has_government = prefeudal_government
+			has_government = high_tribal_government 
+			has_government = subroman_government
```

Comment:
Sub-Roman aren't enable to convert to Feudal through decision

Reply:
Will do.